source/material_model/simpler.cc

```diff
@@ -44,6 +44,8 @@ namespace aspect
       // The Simpler model does not depend on composition
       EquationOfStateOutputs<dim> eos_outputs (1);
 
+      thermal_conductivity.evaluate(in,out);
```

Comment:
A question: The viscosity and eos plugins (and the reaction models as well) compute their properties inside the loop over all points, whereas your conductivity is outside and makes its own loop. Is this on purpose?

This of course has advantages and disadvantages of what one can do with it.